app/questionnaire/rules.py

```diff
@@ -69,6 +69,20 @@ def evaluate_repeat(repeat_rule, answer_store):
         return repeat_function(filtered)
 
 
+def evaluate_skip_condition(skip_condition, answer_store):
```

Comment:
There are 3, it should be possible to consolidate. I started doing it initially and it was a bit of a can of worms so planned on coming back to it later so thanks for the reminder 👍